lms/djangoapps/shoppingcart/tests/test_views.py

```diff
@@ -178,11 +179,12 @@ def _add_course_mode(self, min_price=50, mode_slug='honor', expiration_date=None
         """
         Adds a course mode to the test course.
         """
-        mode = CourseModeFactory.create()
-        mode.course_id = self.course.id
-        mode.min_price = min_price
-        mode.mode_slug = mode_slug
-        mode.expiration_date = expiration_date
+        mode = CourseModeFactory.create(
```

Comment:
No need for `create()` or `save()` below. Simply calling the constructor works the same as `CourseMode.objects.create()` and saves the model to the database.